src/middlewared/middlewared/plugins/virt/license.py

```diff
@@ -0,0 +1,32 @@
+from middlewared.service import private, Service
+
+
+class VirtLicenseGlobalService(Service):
+
+    class Config:
+        namespace = 'virt.global'
+
+    @private
+    async def license_active(self, instance_type=None):
+        """
+        If this is iX enterprise hardware and has NOT been licensed to run virt plugin
+        then this will return False, otherwise this will return true.
+        """
+        system_chassis = await self.middleware.call('truenas.get_chassis_hardware')
+        if system_chassis == 'TRUENAS-UNKNOWN' or 'MINI' in system_chassis:
```

Comment:
We have a `TRUENAS_UNKNOWN` constant defined in the `ixhardware` python module. Please use that instead.